PR-template.md

```diff
@@ -0,0 +1,26 @@
+### Description
+<!--
+_Using one or more sentences, describe the proposed changes and the reason for making them._
+-->
+
+### Related JIRA Issue(s)
+<!--_Please provide the URL(s) for any JIRA issues related to this PR._-->
+
+### Review App URL(s) 
+http://__BRANCH_NAME_HERE__.review.ensembl.org
+
+
+### Knowledge Base
+<!--
+_If there is any new concept which are being implemented then provide link(s) to the Knowledge base (Link to Documentation, Blog etc )_
+-->
+
+### Example(s)
```

Comment:
I think we're avoiding writing tests.
 By adding this part to PRs  we're causing redundancy in OpenAPI spec and again in tests specially when you're confident in your code and you have used OpenAPI tools to test your code.

Reply:
>I think we're avoiding writing tests.

Reply:
So do you think a PR description is a proper place for tests?
An example would be missing the fact that we were returning 500 for an invalid string input in region validation API in this repo. Should we add all the expected responses? 404,500, error messages etc?

Reply:
> So do you think a PR description is a proper place for tests?.....
No, What I think would be good is something like this to have in the PR to give quick overview. Again, This is subjective so We can use this section as new find it appropriate.
![Screenshot 2024-02-27 at 13 34 38](https://github.com/Ensembl/ensembl-web-metadata-api/assets/40833799/e42eedad-83c6-4661-bbfe-79a597a4a6d6)